cpp/data/system_updates/8.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+cd /usr/local/ub_tools/cpp
```

Comment:
DB schema migration must be done anyway, but the data migration in this script should only be done if env variable $TUEFIND_FLAVOUR is ixtheo.

Reply:
Another problem might be that the rss_aggregator table in ub_tools will be dropped in `convert_rss_db_tables` but should be dropped for all systems. Maybe we only leave the lines 4 and 5 out (relbib-related) and leave `convert_rss_db_tables` in. On non-ixtheo systems, the table should be empty anyway.

=> i just checked this; for some reason we had old rows on ub15 (pub_date 2014-2019) but not on sobek, so i truncated the table on ub15 so the migration should not lead to any problems,

Reply:
Done